### DISCUSSION TITLE
Czy?

### DISCUSSION THREAD

**Discussion by jelni** [pxseu/pxseu.com#97] [Q&A] [answered]
Dlaczego!!!?????????????

**Comment by ribxne** [comment, marked as answer]
tak

**Reply by marek12306** [reply]
Wszystkiego najlepszego 
WoT
Walczysz dzielnie w World of Tanks już kolejny rok. Świętujmy w tym wyjątkowym dniu. Przenalizowaliśmy statystyki, by ustalić Twój wyjątkowy styl gry.
MÓJ RAPORT BOJOWY

Odkryj swój specjalny prezent:
Aby uczcić tę rocznicę, oferujemy Ci ekskluzywną zniżkę, którą możesz wykorzystać w sklepie premium.
Rozegraj jedną bitwę losową w ciągu 14 dni, by odebrać swój prezent, który będzie dostępny przez 7 dni.

30% zniżki na wybrane zestawy


CONTACT    Store    Facebook    Twitter    Youtube
Otrzymałeś ten e-mail, ponieważ masz zarejestrowane konto Wargaming.net połączone z tym adresem e-mail. Jeśli nie chcesz dłużej otrzymywać od nas wiadomości, możesz anulować subskrypcję w sekcji Zarządzania kontem. Pamiętaj, że jeżeli Twoje konto Wargaming.net nie było aktywowane, najpierw będziesz musiał je aktywować przez odpowiedni e-mail aktywacyjny, aby otrzymać dostęp do sekcji Zarządzania kontem.

EULA   Politykę prywatności   Centrum obsługi klienta

© 2009-2021 Wargaming.net. Wszystkie prawa zastrzeżone.

**Reply by indexerrowaty** [reply]
> Wszystkiego najlepszego
> WoT
> Walczysz dzielnie w World of Tanks już kolejny rok. Świętujmy w tym wyjątkowym dniu. Przenalizowaliśmy statystyki, by ustalić Twój wyjątkowy styl gry.
> MÓJ RAPORT BOJOWY
> 
> Odkryj swój specjalny prezent:
> Aby uczcić tę rocznicę, oferujemy Ci ekskluzywną zniżkę, którą możesz wykorzystać w sklepie premium.
> Rozegraj jedną bitwę losową w ciągu 14 dni, by odebrać swój prezent, który będzie dostępny przez 7 dni.
> 
> 30% zniżki na wybrane zestawy
> 
> CONTACT Store Facebook Twitter Youtube
> Otrzymałeś ten e-mail, ponieważ masz zarejestrowane konto Wargaming.net połączone z tym adresem e-mail. Jeśli nie chcesz dłużej otrzymywać od nas wiadomości, możesz anulować subskrypcję w sekcji Zarządzania kontem. Pamiętaj, że jeżeli Twoje konto Wargaming.net nie było aktywowane, najpierw będziesz musiał je aktywować przez odpowiedni e-mail aktywacyjny, aby otrzymać dostęp do sekcji Zarządzania kontem.
> 
> EULA Politykę prywatności Centrum obsługi klienta
> 
> © 2009-2021 Wargaming.net. Wszystkie prawa zastrzeżone.

Tej to mail skierowany od Wargaming dla mnie >:(

**Comment by pxseu** [comment]
Tak

**Reply by marek12306** [reply]
Wszystkiego najlepszego 
WoT
Walczysz dzielnie w World of Tanks już kolejny rok. Świętujmy w tym wyjątkowym dniu. Przenalizowaliśmy statystyki, by ustalić Twój wyjątkowy styl gry.
MÓJ RAPORT BOJOWY

Odkryj swój specjalny prezent:
Aby uczcić tę rocznicę, oferujemy Ci ekskluzywną zniżkę, którą możesz wykorzystać w sklepie premium.
Rozegraj jedną bitwę losową w ciągu 14 dni, by odebrać swój prezent, który będzie dostępny przez 7 dni.

30% zniżki na wybrane zestawy


CONTACT    Store    Facebook    Twitter    Youtube
Otrzymałeś ten e-mail, ponieważ masz zarejestrowane konto Wargaming.net połączone z tym adresem e-mail. Jeśli nie chcesz dłużej otrzymywać od nas wiadomości, możesz anulować subskrypcję w sekcji Zarządzania kontem. Pamiętaj, że jeżeli Twoje konto Wargaming.net nie było aktywowane, najpierw będziesz musiał je aktywować przez odpowiedni e-mail aktywacyjny, aby otrzymać dostęp do sekcji Zarządzania kontem.

EULA   Politykę prywatności   Centrum obsługi klienta

© 2009-2021 Wargaming.net. Wszystkie prawa zastrzeżone.

**Comment by indexerrowaty** [comment]
Nie
